src/style.css

```
@@ -1,3 +1,80 @@
+
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+
+  --main-background-color: #fff;
+  --box-shadow-color: #0000000D;
+  --active-link-color: #00ACDC;
+  --default-link-color: black;
+  --hovered-link-color: #00ACDC;
```

Why did you create two variables with the same values?
```suggestion
  --active-link-color: #00ACDC;
  --default-link-color: black;
```